test/test_items_database.py

```diff
@@ -0,0 +1,50 @@
+"""
+Author:  PH01L
+Email:   phoil@osrsbox.com
+Website: https://www.osrsbox.com
+
+Description:
+Tests for module: osrsbox.items_api.all_items
+
+Copyright (c) 2019, PH01L
+
+###############################################################################
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+###############################################################################
+"""
+
+import json
+from pathlib import Path
+
+import jsonschema
+
+
+def test_item_database(path_to_docs_dir: Path):
+    """Unit test to check item database contents against JSON schema
+
+    :param path_to_docs_dir: The path to the `docs` folder.
+    """
+    # Read in the item_schema.json file
+    path_to_schema = Path("test/item_schema.json")
+    with open(path_to_schema, 'r') as f:
```

Comment:
Paths from pathlib are pretty cool in the fact that they remove this boiler plate with `.read_text` https://docs.python.org/3.6/library/pathlib.html#pathlib.Path.read_text

Reply:
Nice hint, I will change this. The more I use pathlib the more I can see why it is such a good library.